bobber/lib/analysis/table.py

```diff
@@ -146,6 +148,38 @@ def fio_iops(results: list) -> Tuple[list, list]:
         return [read, write]
 
 
+def fio_125k_bw(results: list) -> Tuple[list, list]:
+    """
+    Save the FIO 125k bandwidth read and write results.
+
+    Save the read and write results from the FIO 125k bandwidth tests on an
+    increasing per-system basis with the first element in the list being the
+    column header.
+
+    Parameters
+    ----------
+    results : list
+        A ``list`` of ``dictionaries`` containing all results from the tests.
+
+    Returns
+    -------
+    tuple
+        Returns a ``tuple`` of (``list``, ``list``) containing the read and
+        write 125k bandwidth results, respectively.
+    """
+    try:
+        read = [FIO_125K_READ_BW] + [bytes_to_gb(result[1]['125k_bandwidth']
+                                                 ['read'])
+                                     for result in results]
+        write = [FIO_125K_WRITE_BW] + [bytes_to_gb(result[1]['125k_bandwidth']
```

Comment:
Same goes here as above - we will need to investigate this for multi-node results somehow, but my assumption could definitely be wrong.

Reply:
I just copied directly from surrounding fio results - lemme look at the data structure to give you a better answer

Reply:
So the upshot is that I still don't know and would need to print out the data structure to figure it out. Which I can do tomorrow. But I stand by my perspective that if what I've done is broken, the whole scheme here is broken and we should fix it in a separate PR holistically.

Reply:
Here's an example of the structure we're populating:

            '125k_bandwidth': {
                'read': self._average_125k_read_bw(),
                'write': self._average_125k_write_bw(),
                'unit': 'operations/second',
                'parameters': {
                    'read': self._125k_read_bw_params,
                    'write': self._125k_write_bw_params
                }
            },

Given that we're getting data out of this dict lookup and this is iterating through all system counts, it would seem that the result[1] access is skipping some data packed into this dict.

There's a line of code preceding the operation(s) here (that are inside function calls):

    results = sorted(json_results['systems'].items())

Does this sorted function somehow augment the subsequent data structure to pull the data structure's previous index (system count) in as result[0]? That'd be my suspicion, or something along those lines. Because otherwise I'm not understanding what data is at result[0] that we're trying to skip over.

Reply:
Ah, good call. I confused myself on what I was looking at and interpreted the code incorrectly. Turns out that the last line you listed is the important part - the index in the `result` line isn't referring to the system count as I originally thought, but is instead referring to the _value_ of the dictionary as the `results` object is a list of tuples where each tuple is a key: value pair. So, while I agree with you and I'm confident that this will indeed work, it is still confusing to read and I will want to revamp this to make it easier to follow in the future.

Thanks for digging into this!